Added numbers exercise.

danpottshimself opened the pull request:
Numbers exercise completed.

When run, it should display all answers correctly in the console. 

I have added comments to convey the purpose of each code section.

dpotts7 committed:
Added numbers exercise.
I have added comments to convey the purpose of each code section.

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+ var theNumber = 11;
+
+ if (theNumber % 2 ==0){
+ alert("The number " + theNumber + " is even");
```
Don't alert. Ever. Inconsistent with console.

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+//converting celsius to farenheit
+ function convert (){
+ var celc = 12;
+ var faren = ((celc * 2) +30);
```
This is not the correct formula

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+
+//converting celsius to radians
+function convertToRadians () {
+    var celc = 12;
```
celc

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+convertToRadians();
+
+//finding the hypotenuse length
+var perplength = 8;
```
Again, naming

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+var base = 4;
+var oppositeLength= 11;
+var angle = 73;
+var hypoth = ((perplength*perplength) + (base*base));
```
nope = this is the square of the hypotenuse

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+var oppositeLength= 11;
+var angle = 73;
+var hypoth = ((perplength*perplength) + (base*base));
+var hypotenuse = Math.sin(angle)*oppositeLength;
```
Also, this formula is wrong.

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,50 @@
+
+//rolling a six sided dice
+var Die = function(sides) {
+    this.sides = sides || 6;
```
Not creating an object, why 'this.' ?

dpotts7 committed:
Changed naming of variables to full words with correct spelling.

dpotts7 committed:
Fixed formula for celsius conversion.

dpotts7 committed:
no message

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,47 @@
+var base = 4;
+var oppositeLength= 11;
+var angle = 73;
+var hypoth = ((perpLength*perpLength) + (base*base));
```
Again, this is the hypotenuse squared, nor the hypotenuse, or Hypoth (which sounds like a relative of Cthulhu). Either collapse this down to a one-liner, or rename the variable to something less misleading.

keithbarrow commented on exersizes-numbers.js:
```diff
@@ -0,0 +1,47 @@
+//rolling a six sided dice
+
+var roll = function() {
+    var sides = sides || 6;
```
Why? you can't pass in the number of sides so either assume 6 in the function below, or make it passable to the function.

dpotts7 committed:
changed sides to equal 6 as there is no reason to have "sides" as an input when only 6 sides are required.
Reduced code so that the hypotenuse length (newly named instead of hypoth) is now calculated in one line rather than in the console log too.

keithbarrow commented:
:shipit:
